src/broker/cas_schema_info.h

```diff
@@ -32,11 +32,11 @@
 #include "cas_net_buf.h"
 
 #if defined(CAS_FOR_ORACLE) || defined(CAS_FOR_MYSQL)
-#define SCH_STR_LEN	255
+#define SCH_STR_LEN	287
 #define SCH_CLASS_REMARKS_STR_LEN       2048
 #define SCH_REMARKS_STR_LEN     1024
 #else
-#define SCH_STR_LEN	DB_MAX_IDENTIFIER_LENGTH
+#define SCH_STR_LEN	DB_MAX_IDENTIFIER_LENGTH_287
```

Comment:
'DB_MAX_IDENTIFIER_LENGTH_287' is too meaningless.
Can't 'DB_MAX_IDENTIFIER_LENGTH' changed to 287?

Reply:
Remove DB_MAX_IDENTIFIER_LENGTH_287 and only use DB_MAX_IDENTIFIER_LENGTH.
The maximum length of class_name is changed from 255 bytes to 223 bytes.